File: DotNet/CesiumLanguageWriter/Advanced/ICesiumCartesian3ListValuePropertyWriter.cs
using System.Collections.Generic;

namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a list of three-dimensional Cartesian values.
    /// </summary>
    public interface ICesiumCartesian3ListValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value as a list of three-dimensional Cartesian values.
        /// </summary>
        /// <param name="values">The values.</param>
        void WriteCartesian(IEnumerable<Cartesian> values);
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumCartesian3ListValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;

/**
 * A writer that can write a value as a list of three-dimensional Cartesian values.
 */
@SuppressWarnings( {
        "unused",
        "deprecation",
        "serial"
})
public interface ICesiumCartesian3ListValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    * Writes the value as a list of three-dimensional Cartesian values.
    * @param values The values.
    */
    void writeCartesian(Iterable<Cartesian> values);
}